lib/serve-static.module.ts

```diff
@@ -15,14 +15,21 @@ import { AbstractLoader } from './loaders/abstract.loader';
 export class ServeStaticModule implements OnModuleInit {
   constructor(
     @Inject(SERVE_STATIC_MODULE_OPTIONS)
-    private readonly ngOptions: ServeStaticModuleOptions,
+    private readonly ngOptions: ServeStaticModuleOptions[],
     private readonly loader: AbstractLoader,
     private readonly httpAdapterHost: HttpAdapterHost
   ) {}
 
-  public static forRoot(options: ServeStaticModuleOptions = {}): DynamicModule {
-    options.rootPath = options.rootPath || DEFAULT_ROOT_PATH;
-    options.renderPath = options.renderPath || DEFAULT_RENDER_PATH;
+  public static forRoot(
+    _options: ServeStaticModuleOptions | ServeStaticModuleOptions[] = {}
```

Comment:
Please use `_options` back to `options`, since this will be displayed in the intellisense.